library/cwm/src/modules/CWM.rb

```diff
@@ -779,8 +779,10 @@ def handleDebug
     # @param [Array<Hash{String => Object>}] widgets list of widget maps
     # @param [Hash] functions map initialize/save/handle fallbacks if not specified
     #   with the widgets.
+    # @param [Array<Object>] skip_store_for list of events that is cause to not store widgets
+    #   Useful mainly for non-standard redraw of widgets without storing like :reset or :redraw
```

Comment:
list of events for which the value of the widget will not be stored
Useful mainly for non-standard redraw of widgets, like :reset or :redraw